app/src/main/java/com/ataulm/whatsnext/WhatsNextRepository.kt

```diff
@@ -34,13 +37,14 @@ internal class WhatsNextRepository(
         filmSummaries
     }
 
-    suspend fun film(letterboxdId: String) = withContext(Dispatchers.IO) {
+    suspend fun filmSummary(letterboxdId: String) = withContext(Dispatchers.IO) {
         val apiFilm = letterboxdApi.film(letterboxdId)
+        filmSummaryConverter.convert(apiFilm)
+    }
+
+    suspend fun filmRelationship(letterboxdId: String) = withContext(Dispatchers.IO) {
```

Comment:
had to extract this because this only works if the user is signed in, and that wasn't necessarily the case where we were calling `film` before.